src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -1144,14 +1096,20 @@ void GetUserConvCoeffs(EnergyPlusData &state)
 
                 case HcExt::Schedule: { // Schedule
                     ++state.dataSurface->TotExtConvCoeffUser;
-                    auto &userOutConvCoef = state.dataSurface->UserExtConvCoeffs(state.dataSurface->TotExtConvCoeffUser);
-                    userOutConvCoef.SurfaceName = Alphas(1);
-                    userOutConvCoef.WhichSurface = surfNum;
-                    userOutConvCoef.overrideType = OverrideType::Schedule;
-                    userOutConvCoef.ScheduleIndex = ScheduleManager::GetScheduleIndex(state, Alphas(Ptr + 2));
-                    if (userOutConvCoef.ScheduleIndex == 0) {
-                        ShowSevereError(state, format("{}{}=\"{}, invalid value", RoutineName, CurrentModuleObject, Alphas(1)));
-                        ShowContinueError(state, format(" Invalid {} entered={}", ipsc->cAlphaFieldNames(Ptr + 2), Alphas(Ptr + 2)));
+                    auto &userExtConvCoef = state.dataSurface->UserExtConvCoeffs(state.dataSurface->TotExtConvCoeffUser);
+                    userExtConvCoef.SurfaceName = Alphas(1);
+                    userExtConvCoef.WhichSurface = surfNum;
+                    userExtConvCoef.overrideType = OverrideType::Schedule;
+                    userExtConvCoef.ScheduleIndex = ScheduleManager::GetScheduleIndex(state, Alphas(Ptr + 2));
+                    if (userExtConvCoef.ScheduleIndex == 0) {
+                        ShowSevereItemNotFound(state, eoh, ipsc->cAlphaFieldNames(Ptr + 2), Alphas(Ptr + 2));
+                        ErrorsFound = true;
+                    } else if (!ScheduleManager::CheckScheduleValueMinMax(state, userExtConvCoef.ScheduleIndex,
+                                                                          ScheduleManager::Clusive::Inclusive, state.dataHeatBal->LowHConvLimit, // >=
```

Comment:
A new version of this function using enums rather than strings for the comparison options.